src/graphics/webgl/webgl-graphics-device.js

```diff
@@ -309,6 +355,12 @@ class WebglGraphicsDevice extends GraphicsDevice {
         this.initializeRenderState();
         this.initializeContextCaches();
 
+        // start async image bitmap test
+        this.supportsImageBitmap = null;
```

Comment:
should this be initialized to false instead?

Reply:
I thought null would be more useful since it'll evaluate to false, but also indicate the result has not yet been set.

Reply:
fair point